packages/scales/src/compute.ts

```diff
@@ -195,7 +195,7 @@ export const generateSeriesAxis = <Axis extends ScaleAxis, Value extends ScaleVa
 
     switch (scaleSpec.type) {
         case 'linear': {
-            const all = sortBy(uniq(values as number[]), v => v)
+            const all = sortBy(uniq(values as number[]).filter(v => v !== null), v => v)
```

Comment:
I think we should add a unit tests for this, also maybe a comment explaining why we filter out null values, what do you think?